src/SearchFilters.js

```diff
@@ -57,13 +57,19 @@ let BasicSearchFilters = withTheme(
             }
           >
             {setMode && (
-              <DropdownItem onClick={() => setMode('resultsOnly')}>
+              <DropdownItem
+                data-testid="hide-filters"
+                onClick={() => setMode('resultsOnly')}
+              >
                 Hide Filters
               </DropdownItem>
             )}
             <TreePauseButton children={children} Component={DropdownItem} />
             {setMode && !disableAdvancedMode && (
-              <DropdownItem onClick={() => setMode('builder')}>
+              <DropdownItem
+                data-testid="advanced-search"
```

Comment:
We might want a more robust naming convention here - `advanced-search` as a test id could conceivably refer to this button or all of advanced search itself

Reply:
@daedalus28 What about `to-advanced-search`?
This link precedes the advance search itself so they wouldn't be present at the same time. That said, it was vague. It's not a button as much as a link or a drop-down selection really.

Reply:
i don't have a preference as long as there's a clear convention we use everywhere

Reply:
Okay, I did some digging and thinking. What about this convention?
<component name / HTML tag name><purpose><index if present>
There will always be exceptions, but in general, we would try to apply that convention.
In this case, it would be `DropdownItem-linkToAdvancedSearch`
What do you think @daedalus28

Reply:
@daedalus28 I opened an issue for discussion for the naming convention.
https://github.com/smartprocure/spark/issues/12079